internal/jobcontainers/jobcontainer.go

```diff
@@ -577,6 +577,6 @@ func systemProcessInformation() ([]*winapi.SYSTEM_PROCESS_INFORMATION, error) {
 
 // Takes a string and replaces any occurences of CONTAINER_SANDBOX_MOUNT_POINT with where the containers volume is mounted.
 func (c *JobContainer) replaceWithMountPoint(str string) string {
-	str = strings.ReplaceAll(str, "%"+sandboxMountPointEnvVar+"%", c.sandboxMount)
-	return strings.ReplaceAll(str, "$env:"+sandboxMountPointEnvVar, c.sandboxMount)
+	str = strings.ReplaceAll(str, "%"+sandboxMountPointEnvVar+"%", c.sandboxMount[:len(c.sandboxMount)-1])
+	return strings.ReplaceAll(str, "$env:"+sandboxMountPointEnvVar, c.sandboxMount[:len(c.sandboxMount)-1])
```

Comment:
can you just do `c.sandboxMount[:]` for this?

Reply:
yes... fixing 😂

Reply:
Oh actually never mind, early morning brain fart. The reason is because I want to strip the last character intentionally because `c.sandboxMount` has a trailing slash (C:\C\12345678abcdefg\\), so the below would have an extra slash unnecessarily after the env var is replaced. 

%CONTAINER_SANDBOX_MOUNT_POINT%\\path\\to\\thing